App doesnt start after setup

I've clicked the deploy button and deployed it as it was. 

The deplyment was successful, but the app crashed at startup

Logs:
`2020-04-30T15:46:38.688173+00:00 app[web.1]: indentified DB in use postgreSQL
2020-04-30T15:46:38.963195+00:00 app[web.1]: Error: Cannot find module 'request'
2020-04-30T15:46:38.963221+00:00 app[web.1]: Require stack:
2020-04-30T15:46:38.963236+00:00 app[web.1]: - /usr/local/lib/node_modules/n8n/node_modules/request-promise-native/lib/rp.js
2020-04-30T15:46:38.963242+00:00 app[web.1]: - /usr/local/lib/node_modules/n8n/node_modules/n8n-core/dist/src/NodeExecuteFunctions.js
2020-04-30T15:46:38.963242+00:00 app[web.1]: - /usr/local/lib/node_modules/n8n/node_modules/n8n-core/dist/src/index.js
2020-04-30T15:46:38.963243+00:00 app[web.1]: - /usr/local/lib/node_modules/n8n/dist/commands/start.js
2020-04-30T15:46:38.963243+00:00 app[web.1]: - /usr/local/lib/node_modules/n8n/node_modules/@oclif/config/lib/plugin.js
2020-04-30T15:46:38.963243+00:00 app[web.1]: - /usr/local/lib/node_modules/n8n/node_modules/@oclif/config/lib/config.js
2020-04-30T15:46:38.963244+00:00 app[web.1]: - /usr/local/lib/node_modules/n8n/node_modules/@oclif/config/lib/index.js
2020-04-30T15:46:38.963244+00:00 app[web.1]: - /usr/local/lib/node_modules/n8n/node_modules/@oclif/command/lib/command.js
2020-04-30T15:46:38.963244+00:00 app[web.1]: - /usr/local/lib/node_modules/n8n/node_modules/@oclif/command/lib/index.js
2020-04-30T15:46:38.963244+00:00 app[web.1]: - /usr/local/lib/node_modules/n8n/bin/n8n
2020-04-30T15:46:38.963245+00:00 app[web.1]: at /usr/local/lib/node_modules/n8n/node_modules/request-promise-native/lib/rp.js:8:12
2020-04-30T15:46:38.963250+00:00 app[web.1]: at module.exports (/usr/local/lib/node_modules/n8n/node_modules/stealthy-require/lib/index.js:62:23)
2020-04-30T15:46:38.963250+00:00 app[web.1]: at Object.<anonymous> (/usr/local/lib/node_modules/n8n/node_modules/request-promise-native/lib/rp.js:7:15)`

I had the same problem

`2020-05-02T04:19:41.625422+00:00 app[web.1]: Error: Cannot find module 'request'
2020-05-02T04:19:41.625439+00:00 app[web.1]: Require stack:
2020-05-02T04:19:41.625440+00:00 app[web.1]: - /usr/local/lib/node_modules/n8n/node_modules/request-promise-native/lib/rp.js
2020-05-02T04:19:41.625441+00:00 app[web.1]: - /usr/local/lib/node_modules/n8n/node_modules/n8n-core/dist/src/NodeExecuteFunctions.js`

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,7 +1,7 @@
-FROM node:12.16-alpine
+FROM node:lts-alpine
 
 # pass N8N_VERSION Argument while building or use default
-ARG N8N_VERSION=0.62.1
+ARG N8N_VERSION=0.65.0
 
 # Update everything and install needed dependencies
 RUN apk add --update graphicsmagick tzdata

